### PR TITLE
Feature/ft sql

### PR DESCRIPTION
**Pull request by Nicolaswg**
# Main point 🥇 

🦅 
> In this new project we share the command implemented to create a simple database and some queries for getting specific information from the table 😄 

## Most used PSQL commands 👨‍💻 :
- \d
- \db
- \dt

## Most used SQL commands 👨‍💻 :
- SELECT
- FROM
- WHERE
- LIKE
- AND
- OR

> Thanks for taking the time to review my PR hope the project meets all the requirements. Any advice please share it in the comments section 👍

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
-SELECT * from animals WHERE name = 'Luna';
+/* Select elements between 2016 and 2019 */
+SELECT
+  *
+FROM
+  animals
+WHERE
+  date_of_birth BETWEEN '2016-01-01'
+  AND '2019-01-01';
+
```
Comment (mmsesay) [inline queries.sql]: - [ ] This query is good. But what if an animal was born on the 21st March 2019? Currently, your query doesn't cover those animals. You only cover animals born on the 1st day and 1st month of 2016 and 2019.
Please update this query to cover animals born in between the given years.

```sql
SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;
```

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Nicolaswg, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Nicolaswg) [reply queries.sql]: hi, @mmsesay  thanks a lot for your review I didn't think about that case 😵‍💫

**Review by mmsesay — APPROVED** [review]
Hi @Nicolaswg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**